docs/source/upgrading_your_components.md

```diff
@@ -0,0 +1,262 @@
+
+* `ORDERER_CONTAINER`: the name of your ordering node container. Note that you will need to export this variable for each node when upgrading it.
+* `LEDGERS_BACKUP`: the place in your local filesystem where you want to store the ledger being backed up. As you will see below, each node being backed up will have its own subfolder containing its ledger. You will need to create this folder.
+* `IMAGE_TAG`: the Fabric version you are upgrading to. For example, `2.0`.
+
+Note that you will have to set an **image tag** to ensure that the node you are starting using the correct images. The process you use to set the tag will depend on your deployment method.
+
+### Upgrade containers
+
+If you have a native deployment, you should copy your `orderer.yaml` file now. Later, you will need this file to compare against the `orderer.yaml` file that is bundled with the release.
+
+Let’s begin the upgrade process by **bringing down the orderer**:
+
+```
+docker stop $ORDERER_CONTAINER
+```
+
+Once the orderer is down, you'll want to **backup its ledger and MSP**:
+
+```
+docker cp $ORDERER_CONTAINER:/var/hyperledger/production/orderer/ ./$LEDGERS_BACKUP/$ORDERER_CONTAINER
+```
+
+This is where you will need to compare the `orderer.yaml` file bundled with the release against the `orderer.yaml` you copied earlier. A tool like `diff` can be helpful here. Once you have copied over the relevant differences, you can start the node:
+
+Then remove the ordering node container itself (since we will be giving our new container the same name as our old one):
+
+```
+docker rm -f $ORDERER_CONTAINER
+```
+
+This is where you will need to compare the `orderer.yaml` file bundled with the release against the `orderer.yaml` you copied earlier. A tool like `diff` can be helpful here.
+
+Then you can launch the new ordering node container by issuing:
+
+```
+docker run -d -v /opt/backup/$ORDERER_CONTAINER/:/var/hyperledger/production/orderer --name $ORDERER_CONTAINER hyperledger/fabric-orderer:$IMAGE_TAG
+```
+
+Once the ordering node comes up, verify that it is working correctly by querying the ledger. The ordering node should detect that it has been down and pull any blocks it has missed.
+
+## Upgrade the peers
+
+Peers should, like the ordering nodes, be upgraded in a rolling fashion (one at a time). As mentioned during the ordering node upgrade, ordering nodes and peers may be upgraded in parallel, but for the purposes of this tutorial we’ve separated the processes out. At a high level, we will perform the following steps:
+
+1. Stop the peer.
+2. Back up the peer’s ledger and MSP.
+3. Remove chaincode containers and images.
+4. Remove the peer container.
+5. Launch a new peer container using the relevant image tag.
+
+### Set command environment variables
+
+Export the following environment variables before attempting to upgrade your peers.
+
+* `CORE_PEER_CONTAINER`: the name of your peer container. Note that you will need to set this variable for each node.
```

Comment:
Could just call it PEER_CONTAINER to be consistent with ORDERER_CONTAINER above.